config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigtableappprofiles.bigtable.cnrm.cloud.google.com.yaml

```diff
@@ -79,66 +81,78 @@ spec:
+                description: Use a multi-cluster routing policy.
+                properties:
+                  clusterIds:
+                    description: The set of clusters to route to. The order is ignored;
+                      clusters will be tried in order of distance. If left empty,
+                      all clusters are eligible.
+                    items:
+                      type: string
+                    type: array
+                  rowAffinity:
+                    description: Row affinity sticky routing based on the row key
+                      of the request. Requests that span multiple rows are routed
+                      non-deterministically.
+                    type: object
+                type: object
+              name:
+                description: The unique name of the app profile. Values are of the
+                  form `projects/{project}/instances/{instance}/appProfiles/[_a-zA-Z0-9][-_.a-zA-Z0-9]*`.
+                type: string
+              priority:
                 description: |-
-                  If true, read/write requests are routed to the nearest cluster in the instance, and will fail over to the nearest cluster that is available
-                  in the event of transient errors or delays. Clusters in a region are considered equidistant. Choosing this option sacrifices read-your-writes
-                  consistency to improve availability.
-                type: boolean
+                  This field has been deprecated in favor of `standard_isolation.priority`.
+                   If you set this field, `standard_isolation.priority` will be set instead.
+
+                   The priority of requests sent using this app profile.
+                type: string
               resourceID:
-                description: Immutable. Optional. The appProfileId of the resource.
-                  Used for creation and acquisition. When unset, the value of `metadata.name`
-                  is used as the default.
+                description: The BigtableAppProfile name. If not given, the metadata.name
+                  will be used.
                 type: string
               singleClusterRouting:
                 description: Use a single-cluster routing policy.
                 properties:
                   allowTransactionalWrites:
-                    description: |-
-                      If true, CheckAndMutateRow and ReadModifyWriteRow requests are allowed by this app profile.
-                      It is unsafe to send these requests to the same table/row/column in multiple clusters.
+                    description: Whether or not `CheckAndMutateRow` and `ReadModifyWriteRow`
+                      requests are allowed by this app profile. It is unsafe to send
+                      these requests to the same table/row/column in multiple clusters.
                     type: boolean
-                  clusterId:
+                  clusterID:
```

Comment:
I this `clusterID` is more correct. However, since this resource is already in v1beta1, we need to maintain backward compatibility by continuing to use the existing `clusterId`.